package.json

```diff
@@ -107,12 +112,6 @@
     "@types/react-transition-group": "^4.4.4",
     "@types/react-virtualized-auto-sizer": "^1.0.1",
     "@yandex-cloud/axios-wrapper": "^1.0.2",
-    "@yandex-cloud/browserslist-config": "1.0.1",
```

Comment:
We don't need this package any longer?

Reply:
Nope, it was only here because ydb-ui-components required it. I removed this requirement.